src/3d/materials/qgsphongtexturedmaterialsettings.cpp

```diff
@@ -141,7 +141,7 @@ Qt3DRender::QMaterial *QgsPhongTexturedMaterialSettings::toMaterial( QgsMaterial
       Qt3DRender::QShaderProgram *shaderProgram = new Qt3DRender::QShaderProgram();
 
       //Load shader programs
-      const QUrl urlVert( QStringLiteral( "qrc:/shaders/diffuseSpecular.vert" ) );
```

Comment:
diffuseSpecular.vert should be deleted and removed from shaders.qrc

Reply:
Fixed. This was introduced during a rebase.